components/x-podcast-launchers/src/PodcastLaunchers.jsx

```diff
@@ -32,6 +33,16 @@ const rssUrlCopyButtonWrapperStyles = [
 ].join(' ');
 
 
+function defaultFollowButtonRender (conceptId, conceptName, csrfToken, isFollowed) {
```

Comment:
I could be convinced either way here, but although having this as a render prop brings more flexibility, it is unlikely we will actually need it. Especially as this component is being used as part of an experiment, it might better to keep things as simple as possible.

The other slightly confusing thing is that this component will always bring in x-follow-button styles, regardless of whether this render prop is used. What do you think?

Reply:
📝 We need to pass a function to render follow button for ft-app

The reason why ft-app needs own function to render follow buttons.
>We need to add tracking and state management, which is not available on the follow button itself. FT.com is currently using events fired on body to handle these.

Related:
https://github.com/Financial-Times/x-dash/pull/265
https://github.com/Financial-Times/ft-app/blob/569500519fb7ff9e02690541a8cbf5a6110a0e43/lib/components/myft/manage/index.js#L58

Reply:
Thanks @fenglish! Makes total sense 👍